src/ValidatedNumerics.jl

```diff
@@ -4,7 +4,7 @@ __precompile__(true)
 
 module ValidatedNumerics
 
-using CRlibm
+import CRlibm
```

Comment:
There is a bunch of warnings appearing because methods (of CRlibm) are overwritten in `src/intervals/rounding.jl`. I guess this is partially due to this `import`, or is it that some functions are truly overwritten?